assets/src/components/editing/elements/definition/DefinitionToolbar.tsx

```diff
@@ -16,7 +16,7 @@ interface SettingsProps {
 
 export const DefinitionSettings = (props: SettingsProps) => {
   return (
-    <Toolbar context={props.commandContext} orientation="horizontal">
```

Comment:
I'm not sure why that orientation="horizontal" property was working in master, it shouldn't have.